Week-1/HTML Assignment1/index.html

```diff
@@ -0,0 +1,153 @@
+<html>
+  <head>
+	<link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet'>
+    <style>	
+		body {
+			font-family: 'Lato';
+		}
+      .bottomLabel {
+        color: white;
+      }
```

Comment:
move css to external file

Reply:
moved all internal and inline css to external